index.js

```diff
@@ -61,12 +61,9 @@ const compile = function compile(template, compileOpts) {
                 const layoutModule = require.resolve(layoutPath);
                 delete require.cache[layoutModule];
             }
-            Object.keys(require.cache).forEach((module) => {
 
-                if (module === compileOpts.filename) {
-                    delete require.cache[module];
-                }
-            });
```

Comment:
And just like the layout module above, we don't need to loop over the require cache for the view either. 👍